_migration-assistant/migration-phases/replay-captured-traffic.md

```diff
@@ -291,7 +296,7 @@ Migration Assistant creates an Amazon CloudWatch dashboard named `MigrationAssis
 - The number of active connections.
 - The replay speed multiplier. 
 
-You can find the Capture and Replay dashboard in the AWS Management Console for CloudWatch Dashboards in the AWS Region where you deployed Migration Assistant.
+You can find the  Capture-and-Replay  dashboard in the AWS Management Console for CloudWatch Dashboards in the AWS Region where you deployed Migration Assistant.
 
```

Comment:
Line 299: Please confirm technical accuracy. I can't find anything called the "AWS Management Console for Amazon CloudWatch dashboards" ("dashboards" is not capitalized). Do we mean that you can find the Capture-and-Replay dashboard in the Amazon CloudWatch dashboards section of the AWS Management Console?